lib/task-data/base-tasks/copy-key.js

```diff
@@ -0,0 +1,13 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+
+module.exports = {
+    friendlyName: 'Copy Key',
+    injectableName: 'Task.Base.Ssh.CopyKey',
+    runJob: 'Job.CopyKey',
+    requiredOptions: [],
```

Comment:
If a generic (non-node associated) workflow gets run and attempts to use this task, what will happen? The assert for context.nodeID throws and you get a traceback as the workflow error?